no-computer.md

```diff
@@ -46,6 +55,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+Other than the DSL mentioned earlier, I am not aware of standardized ways of describing game rules. Certain game publishers might have specific rules or formatting guides for rule books, but these aren't what this DSL would be about. People have been able to get away with expressing rules in English, so there probably has not been enough of a need for something like this.
```

Comment:
I know one DSL from the first assignment (Josh Kutsko's) was related to this - you might want to check it out. It's called Zillions of Games.